emu_mps/custom_callbacks.py

```diff
@@ -0,0 +1,29 @@
+from emu_mps.mps import MPS
+import torch
+
+
+def calculate_entanglement_entropy(mps: MPS, b: int) -> float:
+    """
+    Calculate the von Neumann entanglement entropy at the bond between sites b-1 and b
+    S_E = -Tr(s^2 * log(s^2)), where s are the singular values at the chosen bond
```

Comment:
```suggestion
    Returns the Von Neumann entanglement entropy of the state `mps` at the bond between sites b-1 and b
         S = -Σᵢsᵢ² log(sᵢ²)), 
    where sᵢ are the singular values at the chosen bond.
```
or something along this line